CreateSnapshot/src/main/java/org/opensearch/migrations/CreateSnapshot.java

```diff
@@ -59,6 +61,10 @@ public static class Args {
             "--s3-role-arn" }, required = false, description = "The role ARN the cluster will assume to write a snapshot to S3")
         public String s3RoleArn;
 
+        @Parameter(names = {
+            "--index-allowlist" }, required = false, description = "A comma separated list of indices to include in the snapshot. If not provided, all indices are included.")
```

Comment:
nit: It would be nice for this to be can you format these to put each annotation parameter its own line (like what's done for CaptureProxy & TrafficReplayer).  Like this
```
        @Parameter(
            required = false,
            names = {"--insecure" },
            arity = 0, description = "Do not check the server's certificate")
        boolean allowInsecureConnections;
```